package.json

```diff
@@ -40,16 +40,12 @@
     "ws": "^7.4.3"
   },
   "scripts": {
-    "file-server": "cd ./file-server && yarn start",
```

Comment:
These scripts are essential for the new architecture. Do not remove them.
`file-server`
`editor`
`editor-server`
`controller-server`